drawBot/context/imageContext.py

```diff
@@ -8,17 +8,28 @@
 from .pdfContext import PDFContext
 
 
+def _nsDataConverter(value):
+    if isinstance(value, AppKit.NSData):
+        return value
+    return AppKit.NSData.dataWithBytes_length_(value, len(value))
+
+def _nsColorConverter(color):
```

Comment:
or reuse a color object from the base context https://github.com/typemytype/drawbot/blob/master/drawBot/context/baseContext.py#L619

this will support directly all possible color tuple combinations `(0, )`,  `(1, .5)`,  `(1, 0, 0)`, `(1, 0, 0, 1)`

Reply:
I can do that. I'm confused about `NSColor.colorWithDeviceRed_green_blue_alpha_()` vs. `NSColor.colorWithCalibratedRed_green_blue_alpha_()`, though. the `Color` object uses the latter, but the former matches `imagePixelColor()` more precisely.